vendors/patterns.yml

```diff
@@ -41,4 +41,42 @@ patterns:
       - name: sendgrid.py
         start_offset: 20
         end_offset: 89
+
+  - name: Okta token
+    type: okta_token
+    regex:
+      version: 0.1
+      pattern: |
+        (0{2}[0-9A-Za-z\-\_]{40})
+      start: |
+        (\A|[^0-9A-Za-z\_\-0x])
+      end: |
+        (\z|[^0-9A-Za-z_=-])
+      additional_not_match:
+        # to cut stuff where there is just numbers
+        - (\d\D|\D\d)
+        # Positive match
+        - [g-zG-Z]
+      additional_match:
+        # Avoid Hexadecimal only 
+        - (^[0-9A-Fa-f\-]+$)
+        # Avoid char only
+        - ([a-zA-Z_-]{20,})
+
+    expected:
+      - name: token.txt
+        start_offset: 12
+        end_offset: 55
+      - name: token.txt
+        start_offset: 69
+        end_offset: 107
+      - name: token.txt
+        start_offset: 121
+        end_offset: 161
+      - name: token.txt
+        start_offset: 174
+        end_offset: 215
+
+    comments:
+      - Okta token, starting with `00` and 40 random alphanumeric
```

Comment:
Could add "plus _ and -" to this comment, and maybe link to their docs where they define it, just for reference?